setup.py

```diff
@@ -10,6 +10,7 @@
     zip_safe=False,
     install_requires=[
         'celery',
+        'dogpile.cache',
```

Comment:
Based on our discussion, we'll also need to add python-memcached as a requirement here.